components/prism-core.js

```diff
@@ -533,21 +533,21 @@ if (script) {
 	_.filename = script.src;
 
 	if (!_.manual && !script.hasAttribute('data-manual')) {
-		_.highlightAutomaticallyCallback = function () {
+		function highlightAutomaticallyCallback() {
```

Comment:
After this change it's not possible to minify script using uglify-js and `"use strict"` declaration:

```
  13971 |
  13972 |       if (!_.manual) {
> 13973 |               function highlightAutomaticallyCallback() {
        |       ^ In strict mode code, functions can only be declared at top level or immediately within another function.
  13974 |                       if (!_.manual) {
  13975 |                               _.highlightAll();
  13976 |                       }
```

Functions shouldn't be declared inside the if code block.